components/log-viewer-webui/server/settings.json

```diff
@@ -11,5 +11,8 @@
     "ClientDir": "../client/dist",
     "StreamFilesDir": "../../../build/clp-package/var/data/streams",
     "StreamTargetUncompressedSize": 134217728,
-    "LogViewerDir": "../yscope-log-viewer/dist"
+    "LogViewerDir": "../yscope-log-viewer/dist",
+
+    "StreamFilesS3Region": "",
+    "StreamFilesS3PathPrefix": ""
```

Comment:
would it make more sense to group these together
- StreamFilesDir
- StreamFilesS3Region
- StreamFilesS3PathPrefix
- StreamTargetUncompressedSize

Reply:
Do you mean put them next to each other in the file?

Reply:
I think Junhao means putting them next to each other.